Gemfile

```diff
@@ -38,6 +38,7 @@ gem 'pg', '~> 1.0.0'
 gem 'prometheus_exporter'
 gem 'puma', '~> 3.12.6'
 gem 'rails', '~> 6.0.3'
+gem 'routing-filter'
```

Comment:
can you add the version, and specify pessimistic locking on patch version explanation here: https://stackoverflow.com/questions/9265213/should-i-specify-exact-versions-in-my-gemfile#:~:text=You%20want%20to%20organize%20your,fixes%2C%20but%20not%20breaking%20changes.

Reply:
done